syntax = "proto3";

import "google/protobuf/wrappers.proto";

package p_config;

service ConfigService {
    rpc StringConfig(PConfigRequest) returns (google.protobuf.StringValue);
}

message PConfigRequest {
    PConfigType config_type = 1;
}

enum PConfigType {
    SECRET_KEY = 0;
}
